features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/.eslintignore

```diff
@@ -1,2 +1,8 @@
 /public/**
-/node_modules/**
\ No newline at end of file
+/node_modules/**
+/source/test/**
+/source/src/app/components/Apis/Details/Security/**
+/source/src/app/components/Apis/Details/Resources/**
+/source/src/app/components/Apis/Details/LifeCycle/**
+/source/src/app/components/Apis/Details/Documents/**
+/source/src/app/components/Apis/Details/Endpoints/**
```

Comment:
@Rajith90  Please review the Endpoint components and remove the eslint exclusion